A job failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/8977385003/job/24656043956) [(plain)](https://github.com/rust-lang-ci/rust/commit/69e9dac7b04b7db4b4c880480ee6dee6707eaef9/checks/24656043956/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
842bef16b8fb: Preparing
064ca6435562: Pushed
4479155cacb3: Pushed
842bef16b8fb: Pushed
##[error]Process completed with exit code 143.
##[error]The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
```

</details>

--- a/src/tools/html-checker/main.rs
+++ b/src/tools/html-checker/main.rs
@@ -29,6 +29,8 @@ fn check_html_file(file: &Path) -> usize {
         .arg("-quiet")
         .arg("--mute-id") // this option is useful in case we want to mute more warnings
         .arg("yes")
+        .arg("--new-blocklevel-tags")
+        .arg("rustdoc-search") // custom elements
         .arg("--mute")
         .arg(&to_mute_s)
         .arg(file);

